src/client/components/post-form.vue

```diff
@@ -398,6 +400,7 @@ export default Vue.extend({
 
 		onChangeUploadings(uploads) {
 			this.$emit('change-uploadings', uploads);
+			if (uploads.length == 0) this.uploading = false;
```

Comment:
```suggestion
			this.uploading = uploads.length > 0;
```

にしたら `@changeUploading="uploading = $event"` 要らないとかない？